docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,40 @@ To get the status of the DNS server:
 ----
 $ minishift dns status
 ----
+
+[[run-against-an-existing-machine]]
+== Run against an Existing Machine
+
+{project} can be run against an existing machine using `vm-driver` as `generic` along with it options.
+To run against an existing machine you first need to configure your (remote/local/VM) machine.
+
+[NOTE]
+====
+This features as of now tested for Fedora/CentOS and RHEL machines.
```

Comment:
s/`This features as of now tested for Fedora/CentOS and RHEL machines.`/ `CentOS/RHEL/Fedora are the suggested Linux distributions for this feature.`/